app/views/users/_search.html.slim

```diff
@@ -2,7 +2,7 @@
   .form-group
     = text_field_tag :search, params[:search], placeholder: "Search user or team", class: "form-control"
   .form-group
-    = submit_tag "Go", class: "form-control btn btn-success"
+    = submit_tag "Search", class: "form-control btn btn-default"
```

Comment:
Well, I am biased of course. Then again, I hate 'Search' on buttons LOL.
If there are no complaints from users, I would vote for reverting.